sources/Makefile

```diff
@@ -243,21 +243,33 @@ workers:
 	@docker compose -f docker-compose.yml build workers
 	@docker compose -f docker-compose.yml up -d workers
 
-# Enable local DB connection to Openshift DEV/TEST/QA
+# Enable local DB connection to Openshift DEV/TEST/QA/STG/PROD
 connect-dev-db:
-	@echo "Forward local port 5454 to Openshift Patroni-master default port."
+	@echo "Forward local port 5454 to Openshift simsdb-pgbouncer default port."
 	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5454."
 	@oc project 0c27fb-dev
-	@oc port-forward services/patroni-master 5454:5432
+	@oc port-forward services/simsdb-pgbouncer 5454:5432
 
 connect-test-db:
-	@echo "Forward local port 5555 to Openshift Patroni-master default port."
+	@echo "Forward local port 5555 to Openshift simsdb-pgbouncer default port."
 	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5555."
 	@oc project 0c27fb-test
-	@oc port-forward services/patroni-master 5555:5432
+	@oc port-forward services/simsdb-pgbouncer 5555:5432
 
 connect-qa-db:
-	@echo "Forward local port 5656 to Openshift Patroni-master default port."
+	@echo "Forward local port 5656 to Openshift simsdb-pgbouncer default port."
 	@echo "Use your prefered DB Manager (e.g. pgAdmin or DBeaver) to connect to the Postgres database using the local port 5656."
 	@oc project 0c27fb-test
-	@oc port-forward services/patroni-master 5656:5432
+	@oc port-forward services/simsdb-pgbouncer 5656:5432
```

Comment:
We no longer have QA DB on `0c27fb-test`. We should either remove `connect-qa-db` or change the `@oc project 0c27fb-test`.

Reply:
Yes i missed it, i believe we may need to remove the connect-qa-db